docs/release-notes/cloud/cloud-2024-04.md

```diff
@@ -0,0 +1,95 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud April 2024.
+included_jira_versions: ['2024.04']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.10.272
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.305
+---
+
+# Cloud April 2024
+
+These release notes are for the Codacy Cloud updates during April 2024.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   Improved [Security and risk management](../../organizations/managing-security-and-risk.md) with a redesigned overview page, offering comprehensive metrics and summary graphs for better visibility and tracking of security findings. An improved findings table now includes a streamlined view of security findings. (TAROT-2546)
+
+    !!! note
+        To align with the new features, all references to "security items" were updated to "security findings".
+
+    ![Security and risk management](../images/tarot-2546.png)
+
+-   Repository admins can now [bypass Codacy status check](../../faq/code-analysis/can-i-bypass-codacy-status-check.md) to unblock the merge of a pull request on <span class="skip-vale">very</span> specific and exceptional situations. (IO-1036)
+-   You can now manually trigger the reanalysis of the coverage of a pull request using the [`reanalyzeCoverage`](https://api.codacy.com/api/api-docs#reanalyzecoverage) API endpoint. (ALA-1003)
+
+## Bug fixes
+
+-   The [Codacy configuration file](../../repositories-configure/codacy-configuration-file.md) no longer overrides the [default ignored files](../../repositories-configure/ignoring-files.md#default-ignored-files). Also, Codacy no longer ignores by default files matching the regular expression `.*docs?/.*`. (IO-1031)
+-   Fixed a parsing error causing detcted issues of a commit to show on the analysis logs instead of listed on the commit's Issues tab. (IO-1054)
```

Comment:
```suggestion
-   Fixed a parsing error causing detected issues of a commit to show on the analysis logs instead of listed on the commit's Issues tab. (IO-1054)
```